src/client/webpack.config.ts

```diff
@@ -36,6 +36,7 @@ module.exports = {
     devServer: {
         port: 9000,
         open: true,
+        historyApiFallback: true,
```

Comment:
Well done! Can you also add a static file that will be used for 404 responses ?

Reply:
Ok I will add